_config_automation.yml

```diff
@@ -14,51 +14,71 @@ data_dest: github
 folder_path:
 
 ###### Calendly ######
-refresh-calendly: yes
+refresh-calendly: no
 calendly_googlesheet:
 
 ###### Citations ######
 refresh-citations: yes
 citation_papers: [
-  "https://scholar.google.com/scholar?cites=6140457238337460780",
-  "https://scholar.google.com/scholar?cites=15945159957067506879" ]
+  "https://scholar.google.com/scholar?cites=9780221468138198188",  #tandfonline paper
+  "https://scholar.google.com/scholar?cites=3064856349813099026" ] #f1000 paper
 citation_googlesheet:
 
 ###### CRAN ######
 refresh-cran: yes
-cran_packages: [ metricminer, ottrpal ]
+cran_packages: [ OCSdata ]
 cran_googlesheet:
 
 ###### GitHub ######
 refresh-github: yes
 github_repos: [
-  fhdsl/metricminer,
-  fhdsl/metricminer-dashboard,
-  jhudsl/OTTR_Template,
-  jhudsl/OTTR_Template_Website
+  opencasestudies/ocs-bp-school-shootings-dashboard, #in OCS Search table
+  opencasestudies/ocs-bp-youth-disconnection, #in OCS Search table
+  opencasestudies/ocs-bp-opioid-rural-urban, #in OCS Search table
+  opencasestudies/ocs-bp-vaping-case-study, #in OCS Search table
+  opencasestudies/ocs-bp-youth-mental-health, #in OCS Search table
+  opencasestudies/ocs-bp-rural-and-urban-obesity, #in OCS Search table
+  opencasestudies/ocs-bp-RTC-wrangling, #in OCS Search table
+  opencasestudies/ocs-bp-RTC-analysis, #in OCS Search table
+  opencasestudies/ocs-bp-co2-emissions, #in OCS Search table
+  opencasestudies/ocs-bp-diet, #in OCS Search table
+  opencasestudies/ocs-bp-air-pollution, #in OCS Search table
+  opencasestudies/ocs-healthexpenditure, #in OCS Search table
+  opencasestudies/OCS_Guide, #Guide for using OCS in teaching
+  opencasestudies/ocs-template, #OCS template repo (this one or the bp one?)
+  opencasestudies/ocs-bp-template #OCS template repo (this one or the non-bp one?)
   ]
 github_googlesheet:
 
 ###### Google Analytics ######
 refresh-ga: yes
-ga_property_ids: [ 422671031, 422558989 ]
+ga_property_ids: [ 256941640, 256923228 ]
 ga_googlesheet:
 
 ###### Google Forms ######
-refresh-googleforms: yes
+refresh-googleforms: no #are there relevant google forms?
```

Comment:
yeah ... there is the main survey - what does metric miner poll from these? (some people identified themselves) - this is on my own account to keep it private, so the credentials will be different - there are some other forms for submissions that have no responses but we could set up the tracking for those - they are with the same OCS account:

https://forms.gle/dppZWygfhUnL6sv48  and
https://forms.gle/V9DxrQ4MCTFAXK1V6 (this one has one response but it was me testing it)

Reply:
Looks like it's collecting the answers (https://htmlpreview.github.io/?https://github.com/fhdsl/metricminer-dashboard/blob/main/docs/googleforms.html), so probably not useful if the forms don't have responses. I'll look and see if there are other things metricminer is capable of tracking for google forms though

Reply:
It can also grab metadata for questions, but that appears to be more of a what types of questions (multiple choice, etc) are in the form. So probably not going to include google forms right now.

Reply:
I think then we either need to adjust it to store the metricminer data in a google sheet, or I can ask Candace if I can make a quick change to metricminer so that number of responses can be returned

Reply:
yeah that might be nice -worth discussing some of the security aspects related to this

Reply:
Do all of the forms require an email address? I see that the two linked above do, and perhaps I can edit the refresh data script to return the number of unique emails?

Reply:
the other form doesnt but that is generally a good idea... I wonder if we can just get the number of rows with anything in them? unfortunately I had to delete the time stamp so we also can't use that for that one

Reply:
I think the number of rows with anything in them is super doable! I can do a stacked PR of what I think the code will be for that.